server/storage/mvcc/key_index.go

```diff
@@ -205,6 +206,18 @@ func (ki *keyIndex) since(lg *zap.Logger, rev int64) []Revision {
 // revision than the given atRev except the largest one (If the largest one is
 // a tombstone, it will not be kept).
```

Comment:
Remove the comment of `(If the largest one is
// a tombstone, it will not be kept).`

Reply:
Done.